### DISCUSSION TITLE
WSA Version 2303 Update!

### DISCUSSION THREAD

**Discussion by hamza-usmani** [microsoft/WSA#277] [Announcements]
We've shipped an update for Windows Subsystem for Android™ on Windows 11 to the [Windows Subsystem for Android™ Preview Program](https://learn.microsoft.com/en-us/windows/android/wsa/preview-program). This update (2303.40000.2.0) will improve reliability, security and enable picture-in-picture (PIP)!

# What’s New
- **Picture-in-picture mode** supported! For apps that use the Android PIP feature, the subsystem now supports this behavior. With the flexibility of having apps in windowed mode and all of the great Windows resize and snap features, this makes multitasking within Android apps even easier

- A new **"Partially running"** system setting added to WSA Settings app, which runs the subsystem with minimal resources but apps launch quicker than "As needed" mode
- Linux kernel updated to 5.15.78
- Improvements to platform reliability
- Android 13 security updates

# Giving feedback
If you are having issues with Windows Subsystem for Android™ – please file feedback via Feedback Hub under Apps > Windows Subsystem for Android™. For more information about troubleshooting and submitting feedback: [Troubleshooting and FAQ for mobile apps on Windows](https://support.microsoft.com/en-us/windows/troubleshooting-and-faq-for-mobile-apps-on-windows-d6062afd-98a1-4018-a7c8-6b3b680a2ea5). If you’re a developer, please give feedback at [our Github site](https://github.com/microsoft/WSA/).

Thanks,
Windows Subsystem for Android™ Team

**Comment by HotCakeX** [comment]
Awesome new features in this update, thank you!
